requirements.txt

```diff
@@ -1,2 +1,7 @@
-flake8 
-
+flake8
```

Comment:
should this be named `requirements.test.txt` as specified in the request?

Reply:
@proppy I thought we could either do that or just add it to `requirements.txt` directly.

Reply:
Do you want me to move it to `requirements.test.txt`?